src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -110,6 +112,14 @@ def create_object(cls, bucket):
         new_object.totalCellCountByOrgan = bucket['cell_count']['value']
         return new_object
 
+    @classmethod
+    def create_object_from_simple_count(cls, count):
+        new_object = cls()
```

Comment:
Consider renaming `new_object` to `self`

Reply:
The other methods in this file all use `new_object`.  Should those be renamed too?

Reply:
Nope. Consistency trumps. Leave as is.